libs/platform/src/lib/utils/styles/_variables.scss

```diff
@@ -0,0 +1,77 @@
+// Variables
```

Comment:
none of this is themable

Reply:
> @sKudum the PR is classified as breaking changes. Is this the case?

No Deno, all css classes and Variables used are independent there are not editing already existing css of fundamental-styles

Reply:
> none of this is themable

I am sorry, I did not get your comment. Could you please elaborate.